lib/compiler/static-analysis/scjson-analyzer.js

```diff
@@ -69,8 +69,39 @@ var scJsonAnalyzer = {
 		}
 
 		var actionTags = {
-			'data': function (node) {
-				return validateJavascriptAssignment(node.id, node.expr);
+			'data': function (node, done) {
+				//If there is an external file and
+				//src attribute starting exactly with "file:"
+				if(node.src && node.src.indexOf('file:') === 0) {
+					asyncStarted();
+
+					getFileContents(node.src.substring(5), function (error, content) {
+						if(error) {
+							done([error]);
+						} else {
+							delete node.src;
+							node.expr = { 	$column: node.$column,
+											$line: node.$line,
+											expr: "\"" + normalizeWhitespace(content.toString()) + "\""
```

Comment:
If the string contains a double quote this will totally break. You should just JSON.stringify on content.toString() to escape it.